css.md

```diff
@@ -137,3 +138,19 @@ $_componentNameColor: #000;
 ```scss
 $_componentNameColor: $color1;
 ```
+
+### [2.3](#scss--comments): Comments
+
+When commenting in sass be sure to use `//` version of comments this version will be stripped out during the compilation procress
+
+```scss
+//Bad
+
+ /*this is a bad comment */
```

Comment:
These comments seem redundant since there is a labeling comment above each that already says `//bad` or `//good`. Maybe comments like `/*this will remain in the compiled stylesheet*/` and `//this will be stripped out of the compiled stylesheet` would work better?